ovs_performance.py

```diff
@@ -1662,7 +1594,7 @@ def get_traffic_rx_stats_from_vm(vm, **kwargs):
     result = dut_shell.dut_exec('', raw_cmd=['sh', '-c', cmd], die_on_error=True)
 
     pkt_rates = [int(re.sub(r'^\s*Rx-pps:\s*', '', s))
-                 for s in re.findall(r'^\s*Rx-pps:\s*\d+', result.stdout_output,
+                 for s in re.findall(r'^\s*Rx-pps:\s*\\d+', result.stdout_output,
```

Comment:
This one does not need to change as it's an r''. It was causing my tests to fail...